### PR TITLE
grt: fix duplicated points on edges

### PR DESCRIPTION
**Pull request by eder-matheus**
Fixes OR-1652

### CODE REVIEW

**Review by maliberty — COMMENTED** [review]
Looks ok.  Have you done a secure CI?

**Comment by eder-matheus** [comment]
> Looks ok. Have you done a secure CI?

Public CI finished with these two failures:
```
asap7 aes (base)
 Flow reached last stage.
 Found 1 metrics failures.
     [ERROR] finish__timing__setup__ws fail test: -24.3564 >= -20.0

asap7 sha3 (base)
 Flow reached last stage.
 Found 2 metrics failures.
     [ERROR] finish__timing__setup__ws fail test: -127.984 >= -32.5
     [ERROR] finish__timing__wns_percent_delay fail test: -17.950438 >= -13.53
```

Waiting for the private ci now.

**Comment by eder-matheus** [comment]
@maliberty After merging the latest master branch, public and secure ci are green.

**Review by maliberty — APPROVED** [review]

**Pull request merged**
